src/Deserializers/SnakDeserializer.js

```diff
@@ -35,7 +35,7 @@ MODULE.SnakDeserializer = util.inherit( 'WbSnakDeserializer', PARENT, {
 			try {
 				dataValue = dv.newDataValue( type, value );
 			} catch( error ) {
-				dataValue = new dv.UnDeserializableValue( value, type, error );
+				dataValue = new dv.UnDeserializableValue( type, value, error.message );
```

Comment:
For reference: Order changed in https://github.com/wmde/DataValuesJavascript/pull/58, type will change in https://github.com/wmde/DataValuesJavascript/pull/78, both to be released as 0.7.0.